run_tests.sh

```diff
@@ -0,0 +1,3 @@
+alienv enter o2codechecker/latest GCC-Toolchain/latest; 
+O2codecheck ex.cpp -fix --checks=-*,alice* --
```

Comment:
My bad. This file was not meant to be uploaded, because it is not done. Its intent was to run all tests and it should be in the **o2codechecker\test\** directory.

I will remove it.